main/acle.md

```diff
@@ -4746,6 +4764,39 @@ stored to memory is modified by replacing the low 32 bits of
 `value.val[0]` with the contents of the `ACCDATA_EL1` system register.
 The returned value is the same as for `__arm_st64bv`.
 
+## Atomic store with PCDPHINT intrinsics
+
+These intrinsics provide an atomic store intrinsic, which will
+make use of the `STSHH` hint instruction immediately followed by the
+associated store instruction.
+This intrinsic is available when `__ARM_FEATURE_PCDPHINT` is defined.
+
+To access these intrinsics, `<arm_acle.h>` should be included.
+
+``` c
+  // Variants are also available for:
+  // _s8, _u16, _s16, _u32, _s32, _u64, _s64, _u128,
+  // _mf8, _bf16, _f16, _f32, _f64
+  void __arm_stshh_u8(uint8_t *addr,
+                      uint8_t data,
+                      int mem, /* Memory order */
+                      int ret); /* Retention Policy */
+```
+
+The first argument in these intrinsics is a pointer `addr` containing an address.
+The second argument `data` is the data which is to be stored.
+The 3rd and 4th arguments can contain the following values.
+
+| **Memory Order**     | **Value** | **Summary**                                                                                     |
+| ---------------------| --------- | ----------------------------------------------------------------------------------------------- |
+| __ATOMIC_RELAXED     | 0         | No constraints imposed on other reads or writes, only this operation's atomicity is guaranteed. |
```

Comment:
Do we need to specify the actual numeric values? Are we also specifying an ABI here?